protocol/requests/create_account.go

```diff
@@ -32,6 +32,10 @@ type CreateAccount struct {
 }
 
 func (c *CreateAccount) Validate() error {
+	return ValidateAccountCreationRequest(*c)
+}
+
+func ValidateAccountCreationRequest(c CreateAccount) error {
 	// TODO(cammellos): Add proper validation for password/displayname/etc
```

Comment:
does this TODO have an issue alongside it?

Reply:
No, but I can create one